src/eva/vision/data/datasets/wsi.py

```diff
@@ -72,7 +72,7 @@ def _coords(self) -> wsi.PatchCoordinates:
     def __getitem__(self, index: int) -> torch.Tensor:
         x, y = self._coords.x_y[index]
         width, height, level_idx = self._coords.width, self._coords.height, self._coords.level_idx
-        patch = self._wsi.read_region((x, y), (width, height), level_idx)
+        patch = self._wsi.read_region((x, y), level_idx, (width, height))
```

Comment:
here its good to add the kwargs names, like `self._wsi.read_region((x, y), level=level_idx, size=(width, height))`